src/frontend/components/node-item/node-attributes.html

```diff
@@ -1,2 +1,9 @@
-<span class="node-item-description" *ngIf="attributes.length > 0">
-  (<template ngFor let-d [ngForOf]="attributes" let-last="last"><p class="node-item-property">{{d.key}}</p><span class="punctuation">=&quot;</span><p class="node-item-value">{{d.value}}</p><span class="punctuation">&quot;</span><span *ngIf="!last" class="punctuation">,&nbsp;</span></template>)</span>
\ No newline at end of file
+<span *ngIf="attributes.length > 0">
+  <span class="node-item-description" [ngClass]="{hide: !show}">
```

Comment:
Do we need to create a new class for visibility, can you not use " [hidden]" which hide/show an element.